src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_Opacity.cs

```diff
@@ -48,27 +48,27 @@ public async Task When_Opacity()
 
 			await renderer.RenderAsync(SUT);
 
-			var si = new RawBitmap(renderer, SUT);
+			var si = await RawBitmap.From(renderer, SUT);
 
 			var width = SUT.tbOpacity1_0.ActualWidth;
 			var height = SUT.tbOpacity1_0.ActualHeight;
-			await ImageAssert.HasColorAtChild(si, SUT.tbOpacity1_0, (width / 4) * 3, height / 2, Colors.Black);
+			ImageAssert.HasColorAtChild(si, SUT.tbOpacity1_0, (width / 4) * 3, height / 2, Colors.Black);
 
 			width = SUT.tbOpacity0_5.ActualWidth;
 			height = SUT.tbOpacity0_5.ActualHeight;
-			await ImageAssert.HasColorAtChild(si, SUT.tbOpacity0_5, (width / 4) * 3, height / 2, "#FF808080");
+			ImageAssert.HasColorAtChild(si, SUT.tbOpacity0_5, (width / 4) * 3, height / 2, "#FF808080");
 
 			width = SUT.tbOpacity0_1.ActualWidth;
 			height = SUT.tbOpacity0_1.ActualHeight;
-			await ImageAssert.HasColorAtChild(si, SUT.tbOpacity0_1, (width / 4) * 3, height / 2, "#0xFFE6E6E6");
+			ImageAssert.HasColorAtChild(si, SUT.tbOpacity0_1, (width / 4) * 3, height / 2, "#0xFFE6E6E6");
```

Comment:
@amelield I took a look at this failure. The test is failing because of the extra `0x`. It should be `#0xFFE6E6E6`.

The color parsing ends up here:

https://github.com/unoplatform/uno/blob/1e9a8111358df0eb47cfb92f21c7a85c42efd9a1/src/Uno.UWP/UI/Colors.cs#L117-L123

It's very very very weird this code path is returning Red. What's special with Red to be a "fallback" in this error scenario?

The reason why this has failed in this PR but not in master is we never did the assertion before. The `HasColorAtChild` method was returning a Task, but we never `await`ed it.

Reply:
Thank you very much I will remove the 0x from the test. Finger crossed